test/test_collection.py

```diff
@@ -182,6 +182,14 @@ def test_estimated_document_count(self):
         last_cmd_payload = self.explain.last_cmd_payload
         self._compare_command_dicts(last_cmd_payload, last_logger_payload)
 
+    def test_cli_tool(self):
+        import subprocess
+        res = subprocess.run(["python3",  "-m", "pymongoexplain",
+                              "/home/travis/build/mongodb-labs/pymongoexplain/test/testscript.py"])
```

Comment:
Instead of using a path that only works on travis and nowhere else (this test can never pass on my mac, for example), use the relative locations of the test script and and this test file to figure out what path to pass.

You can find the abspath of the directory in which the test file resides as follows: 
`os.path.dirname(os.path.realpath(__file__))`
Then append to this path (`os.path.join`) the name of the test script and pass the resulting path to `subprocess.run`.

Reply:
Done.